### ISSUE TITLE
preobfuscate won't run

### ISSUE THREAD

**Issue by justinbui** [EmpireProject/Empire#675]
## Empire Version
Empire 2.1

## OS Information (Linux flavor, Python version)
Debian Stable, Python 2.7.9

## Expected behavior and description of the error, including any actions taken immediately prior to the error. The more detail the better.
When running 'preobfuscate' or 'preobfuscate module' it is expected to run Invoke-Obfuscation on all modules or the specific module. I'm encountering an error and sent back to the default Empire screen.

Actions taken prior: None

## Screenshot of error, embedded text output, or Pastebin link to the error
https://pastebin.com/3dzNMTG6

## Any additional information
I've confirmed that powershell is in my path and that Invoke-Obfuscation is in my PS path.

**Comment by cobbr** [comment]
Try on latest master, think it should be fixed.

**Comment by justinbui** [comment]
Thanks, the latest master fixed it!

Some seg faults/PS errors on some modules but most of them worked.

**Comment by cobbr** [comment]
I'd be interested to know the errors you are running into. Mind posting screenshots of the errors, along with your PS version?

**Comment by justinbui** [comment]
Here's the output from 'powershell --version'
- powershell v6.0.0-beta.6

I just tried preobfuscating again, I didn't get the PS errors anymore but still running into some seg faults
https://pastebin.com/uUdJmQAz

**Comment by cobbr** [comment]
Try:
```
$ apt-get remove powershell
$ ./setup/install.sh
```

There's a bug in PowerShell running on Debian 9/Kali that seg faults when it attempts to talk to the telemetry server. The new install.sh script installs powershell and disables telemetry, which should prevent the seg faults.

**Comment by justinbui** [comment]
Thanks for the help cobbr.

After removing powershell and rerunning the install script the seg faults are no more!

I only ran into one powershell error when trying to obfuscate Invoke-Kerberoast. I've attached a link: https://pastebin.com/13HQEQxV

The error I see repeated is "Attribute argument must be a constant or a script block."

The rest of the modules obfuscated fine though, using the default obfuscation.
